pom.xml

```diff
@@ -22,32 +22,51 @@ limitations under the License. -->
 
   <name>BlazingMQ Java SDK Bundle</name>
   <description>BlazingMQ Java SDK Bundle</description>
+  <url>https://github.com/bloomberg/blazingmq-sdk-java</url>
   <organization>
     <name>Bloomberg L.P.</name>
+    <url>https://www.bloomberg.com</url>
   </organization>
 
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <id>bloomberg-oss</id>
+      <name>BlazingMQ Contributors</name>
+      <email>opensource@bloomberg.net</email>
+      <organization>Bloomberg L.P.</organization>
+      <organizationUrl>https://github.com/bloomberg</organizationUrl>
+    </developer>
+  </developers>
+
   <modules>
     <module>bmq-sdk</module>
     <module>bmq-examples</module>
   </modules>
 
   <scm>
-    <developerConnection>scm:git:ssh://${git.url}</developerConnection>
+    <connection>git@github.com:bloomberg/blazingmq-sdk-java.git</connection>
+    <developerConnection>git@github.com:bloomberg/blazingmq-sdk-java.git</developerConnection>
     <tag>HEAD</tag>
+    <url>https://github.com/bloomberg/blazingmq-sdk-java</url>
   </scm>
 
   <distributionManagement>
 
     <repository>
-      <id>central</id>
-      <name>Artifactory (Deployable) Central Artifacts Repository</name>
-      <url>${artifactory.central.url}</url>
+      <id>ossrh</id>
+      <url>https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/</url>
```

Comment:
I am assuming this is the right endpoint?

Reply:
It is mentioned here:
https://central.sonatype.org/publish/publish-guide/#releasing-to-central

```
Note: As of February 2021, all new projects began being provisioned on https://s01.oss.sonatype.org/. If your project is not provisioned on https://s01.oss.sonatype.org/, please login to the legacy host https://oss.sonatype.org/.
```

Reply:
Before this change, endpoints were like this:

https://github.com/simpligility/ossrh-demo/blob/a1a78460ae500c3f605f4175bb358c8b15db5da8/pom.xml#L32-L41

```
 <distributionManagement>
  <snapshotRepository>
   <id>ossrh</id>
   <url>https://oss.sonatype.org/content/repositories/snapshots</url>
  </snapshotRepository>
  <repository>
   <id>ossrh</id>
   <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
  </repository>
 </distributionManagement>
```